Profiler/Profiler.example.yml

```diff
@@ -23,6 +23,10 @@ match:
       # Light mode is recommended and enabled by default.
       # Turn it off if you use ngen.exe for your assemblies
       light_mode: true
+      # Enable eager writing of traces after the specified amount
+      # of method calls (i.e. write to disk immediately).
+      # This should only be used in conjunction with light mode.
+      eagerness: 0
```

Comment:
Can you please add a use case, so users know when this should be used? IMO the only valid use case is if the profiler is killed before it can finish writing the trace files